package.json

```diff
@@ -29,18 +27,22 @@
     "bigtable"
   ],
   "scripts": {
-    "cover": "nyc --reporter=lcov mocha test/*.js && nyc report",
     "docs": "jsdoc -c .jsdoc.js",
     "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "test-no-cover": "mocha test/*.js",
-    "test": "npm run cover",
+    "test": "nyc mocha build/test",
     "generate-scaffolding": "repo-tools generate all && repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
     "lint": "eslint '**/*.js'",
-    "system-test": "mocha system-test/*.js --timeout 600000",
+    "presystem-test": "npm run compile",
+    "system-test": "mocha build/system-test --timeout 600000",
     "snippet-test": "mocha samples/document-snippets/tests/*.js --timeout 600000",
     "fix": "eslint --fix '**/*.js'",
     "docs-test": "linkinator docs -r --skip www.googleapis.com",
-    "predocs-test": "npm run docs"
+    "predocs-test": "npm run docs",
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r src/v2 build/src/ && cp -r proto* build/",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "proto": "mkdir -p proto && pbjs -t static-module -w commonjs -p node_modules/google-proto-files google/bigtable/v2/bigtable.proto | pbts -o proto/bigtable.d.ts -"
```

Comment:
I think we want to build from `protos` instead of `node_modules/google-proto-files`